src/components/class-work.tsx

```diff
@@ -0,0 +1,79 @@
+import { inject, observer } from "mobx-react";
+import * as React from "react";
+
+import "./my-work.sass";
```

Comment:
nit: our convention elsewhere has been to put the `.sass` import last among the imports.